Over21/controllers/ViewController.swift

```diff
@@ -322,25 +304,73 @@ extension ViewController: CaptureHelperDeviceDecodedDataDelegate {
         }
     }
     
-    private func checkIfUserIsOver21() {
-        if let dateOfBirth = map["DBB"], let expiryDate = map["DBA"] {
-            if let cardExpiryDate = dateFormatter.date(from: expiryDate) {
+    private func parseDecodedBarCode(_ data: String) {
+        if data.starts(with: "@") {
+            let words = data.components(separatedBy: "\n")
```

Comment:
we should split this method in 2. One for the driver license parsing and one for the other format parsing

Reply:
done split parse method in 2